setup.py

```diff
@@ -23,8 +23,9 @@
 def get_version():
     version_file = "src/PIL/_version.py"
     with open(version_file, encoding="utf-8") as f:
-        exec(compile(f.read(), version_file, "exec"))
-    return locals()["__version__"]
+        lcl = {}
+        exec(compile(f.read(), version_file, "exec"), lcl)
```

Comment:
`lcl` is the second argument to `exec()`. Looking at https://docs.python.org/3.13/library/functions.html#exec, this would be `globals`, right? So maybe `lcl` is a confusing name, because it makes one think that `locals` is being used instead?